src/components/datatables/Actions.tsx

```diff
@@ -29,24 +33,42 @@ interface Props extends CommonProps {
   rightSide?: ReactNode;
   onFilterChange?: any;
   onStatusChange?: any;
+  onCustomFilterChange?: any;
```

Comment:
Can we typehint this? `any` isn't actually helpful, and I have plans to implement no implicit any rule so this will throw an error in the future.

Reply:
Yeah, all types are adjusted now, including type for `onFilterChange` prop.